prusti-tests/tests/parse/fail/ghost-constraints/trait-bounds-illegal-4.rs

```diff
@@ -0,0 +1,14 @@
+use prusti_contracts::*;
+
+trait A { }
+trait B { }
+
+#[ghost_constraint(T: Fn(A) -> B, [ //~ ERROR: unexpected Prusti syntax
```

Comment:
I'm not sure I understand why this should fail.

Reply:
Ah good point. It should not fail but it currently does because the parser can't handle it.
I will remove the test.